### PR TITLE
Add Deno Runtime Icon

### LINKED ISSUE #65697: Add Deno Icon Runtime Icon
### Problem Statement

When we have an issue occuring on a server with a Deno runtime, there is a question mark for the icon. While there is no question it is in fact on Deno, and we know it :D 

and there are nice icons we can show instead 🦖 

### Solution Brainstorm

Let's show the icon! 🦕 

we already have it here https://github.com/getsentry/platformicons/blob/master/svg/deno.svg

I don't know hwo it works with this paltform icons repo if this is also connected to use in runtime or that's something else

### Product Area

Issues

### PR DESCRIPTION
**Pull request by smeubank**
QoL, this has been bugging me

Should show the Deno icon, from Deno runtimes hosted on Supabase and others

after:
image

image

### CODE REVIEW

**Review by evanpurkhiser — APPROVED** [review]

File: src/sentry/static/sentry/images/logos/logo-deno.svg
Comment (evanpurkhiser) [inline src/sentry/static/sentry/images/logos/logo-deno.svg:1]: I think this image should be edge to edge. Cc @doralchan

Comment (evanpurkhiser) [inline src/sentry/static/sentry/images/logos/logo-deno.svg:1]: Also, this appears to have a white background.

**Review by evanpurkhiser — CHANGES_REQUESTED** [review]
Icon needs a few adjustments

File: src/sentry/static/sentry/images/logos/logo-deno.svg
Comment (evanpurkhiser) [reply src/sentry/static/sentry/images/logos/logo-deno.svg:1]: Let's also run it through https://jakearchibald.github.io/svgomg/

Comment (evanpurkhiser) [reply src/sentry/static/sentry/images/logos/logo-deno.svg:1]: Try this instead


```
<svg width="80" height="80" fill="none"><path fill="#000" d="M40 0a40 40 0 1 1 0 80 40 40 0 0 1 0-80Z"/><path fill="#fff" d="M17.697 14.788a1.94 1.94 0 0 1 3.733.97v.097l-2.666 9.842a1.952 1.952 0 0 1-3.782-.97v-.048l2.666-9.843v-.048h.049Zm34.085 1.842a1.959 1.959 0 0 1 3.782 1.018v.049l-2.134 7.758-.145-.146a20.776 20.776 0 0 0-3.297-2.036l1.794-6.594v-.049ZM33.067 4.752c1.26-.243 2.472-.437 3.733-.534a1.94 1.94 0 0 1 0 .873l-2.667 9.89a1.952 1.952 0 1 1-3.782-.969v-.048l2.425-9.212h.29Zm27.636 8.194.485-1.94.29.146a39.492 39.492 0 0 1 3.055 2.618l-.048.194a1.952 1.952 0 0 1-3.782-.97v-.049ZM44.848 8l.97-3.49.243.048c1.212.194 2.375.484 3.539.824l-.97 3.636v.049a1.952 1.952 0 1 1-3.782-.97V8Zm-8.145 52.897a1.94 1.94 0 0 1 3.636-.34v.291l.146.485.048.291v.049c.005.08.005.161 0 .242l-.048.049-2.667 9.842v.049a1.952 1.952 0 1 1-3.782-.97v-.097l2.667-9.843v-.048Zm-16.242-11.2c.042-.084.09-.165.145-.242.97.824 2.085 1.6 3.394 2.278l-2.424 8.922v.048a1.952 1.952 0 0 1-3.782-.97v-.097l2.667-9.842v-.097Zm-8.582-10.23a1.952 1.952 0 1 1 3.782.97v.096l-2.667 9.843a1.952 1.952 0 1 1-3.782-.97v-.048l2.667-9.843v-.048Zm59.54-2.28a1.958 1.958 0 1 1 3.781 1.02v.047l-2.667 9.843v.048a1.952 1.952 0 0 1-3.782-.97v-.096l2.667-9.843v-.048ZM4.314 35.83c.485-4.606 1.94-9.066 4.17-13.09l.485.048a1.94 1.94 0 0 1 1.357 2.376l-2.666 9.89a1.94 1.94 0 0 1-3.394.776h.048Zm60.412-11.684a1.94 1.94 0 0 1 3.734.97v.096l-2.667 9.842a1.952 1.952 0 1 1-3.782-.97v-.048l2.667-9.842v-.049h.048ZM25.842 65.697a1.952 1.952 0 0 1 3.782.97v.048l-1.94 7.079-.242-.097a35.884 35.884 0 0 1-3.393-1.455l1.793-6.545Z"/><path fill="#fff" fill-rule="evenodd" d="M38.449 22.642c-11.637 0-20.703 7.322-20.703 16.437 0 8.582 8.339 14.06 21.236 13.818 1.115 0 1.454.727 1.794 1.697.29.824.97 4.654 1.697 8.242l.388 1.988.097.485c.63 3.442 1.309 6.885 1.794 10.327A36.17 36.17 0 0 0 66.666 64.05l-6.06-22.691c-1.552-5.48-3.298-10.522-8-14.206a22.498 22.498 0 0 0-14.158-4.51Zm2.472 6.594a2.521 2.521 0 1 1 .698 4.995 2.521 2.521 0 0 1-.698-4.995Z" clip-rule="evenodd"/></svg>
```

We'll need to add a dark mode exception for this as well

**Comment by Jesse-Box** [comment]
@smeubank @evanpurkhiser 

logo-deno

```
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 80 80"><path d="M40 0c22.092 0 40 17.909 40 40 0 22.092-17.908 40-40 40C17.909 80 0 62.092 0 40 0 17.909 17.909 0 40 0Z"/><path fill="#fff" d="M17.653 14.771a1.953 1.953 0 0 1 3.77 1.018l-.012.047-2.656 9.844-.013.047a1.953 1.953 0 0 1-3.77-1.017l.012-.048 2.656-9.843.013-.048ZM51.783 16.632a1.953 1.953 0 0 1 3.77 1.018l-.012.047-2.087 7.734-.172-.129a21.865 21.865 0 0 0-3.297-2.009l1.785-6.614.013-.047ZM33.078 4.729a35.916 35.916 0 0 1 3.713-.525c.065.29.061.592-.01.88l-.012.048-2.656 9.844-.013.047a1.953 1.953 0 0 1-3.77-1.018l.012-.047 2.476-9.178.26-.051ZM60.712 12.938l.521-1.934.231.17a36.12 36.12 0 0 1 3.076 2.572l-.057.21-.013.047a1.953 1.953 0 0 1-3.77-1.018l.012-.047ZM44.837 8.016l.942-3.491.278.046a35.72 35.72 0 0 1 3.543.788l-.992 3.675-.013.047a1.953 1.953 0 0 1-3.77-1.018l.012-.047ZM36.721 60.902a1.954 1.954 0 0 1 3.606-.344l.058.286.092.46.063.318-.008.075c-.009.074-.022.148-.04.222l-.013.048-2.656 9.843-.013.047a1.953 1.953 0 0 1-3.77-1.017l.012-.047 2.656-9.844.013-.047ZM20.47 49.74c.034-.107.075-.209.123-.306a17.66 17.66 0 0 0 3.383 2.306l-2.409 8.925-.013.047a1.953 1.953 0 0 1-3.77-1.017l.012-.048 2.656-9.843.019-.065ZM11.878 39.444a1.953 1.953 0 0 1 3.77 1.017l-.012.048-2.657 9.844-.013.047a1.953 1.953 0 0 1-3.77-1.018l.012-.047 2.656-9.844.014-.047ZM71.437 37.209a1.953 1.953 0 0 1 3.77 1.017l-.012.047-2.656 9.844-.013.047a1.953 1.953 0 0 1-3.77-1.017l.012-.047 2.656-9.844.013-.047ZM4.303 35.821a35.688 35.688 0 0 1 4.171-13.087 1.953 1.953 0 0 1 1.85 2.415l-.012.047-2.656 9.844-.013.047a1.953 1.953 0 0 1-3.34.734ZM64.71 24.132a1.953 1.953 0 0 1 3.77 1.017l-.013.047-2.656 9.844-.013.047a1.953 1.953 0 0 1-3.77-1.017l.012-.048 2.656-9.843.013-.047ZM25.842 65.668a1.954 1.954 0 0 1 3.77 1.017l-.012.047-1.9 7.045-.266-.098a35.7 35.7 0 0 1-3.362-1.455l1.757-6.51.013-.046Z"/><path fill="#fff" fill-rule="evenodd" d="M38.438 22.642c-11.632 0-20.68 7.323-20.68 16.43 0 8.6 8.325 14.091 21.232 13.816 1.097-.023 1.414.714 1.771 1.708.3.834 1.026 4.662 1.704 8.24v.003l.379 1.992.099.517c.652 3.414 1.307 6.84 1.813 10.278 8.428-1.115 16.22-5.277 21.93-11.556l-6.09-22.713c-1.518-5.486-3.292-10.54-7.987-14.212-3.778-2.954-8.577-4.503-14.172-4.503Zm2.5 6.577a2.5 2.5 0 1 1 0 5 2.5 2.5 0 0 1 0-5Z" clip-rule="evenodd"/></svg>
```

**Review by evanpurkhiser — APPROVED** [review]

**Pull request merged**